docs/source/_src_user_guide/state_propagation/running_simulation/propagation_results.rst

```diff
@@ -0,0 +1,125 @@
+Propagation results
+===================
+
+After numerically propagating the dynamics, as follows:
+
+.. code-block:: python
+
+    dynamics_simulator = numerical_simulation.create_dynamics_simulator(
+        body_system, propagator_settings)
+    propagation_results = dynamics_simulator.propagation_results
+
+the results are stored in the
+``propagation_results`` object. For a single-arc propagation, this object is of the type
+:class:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults`.
+Multi- and hybrid-arc results are discussed briefly :ref:`below <multi_hybrid_arc_results>`.
+
+The three primary numerical results of the propagation are:
+
+- The numerical results of the propagation, in 'conventional' coordinates
+  (e.g. Cartesian elements for translational dynamics, see :ref:`conventional_states`), stored in the
+  :attr:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults.dependent_variable_history.state_history` attribute
+- The numerical results of the propagation, in the propagated coordinates, stored in the
+  :attr:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults.dependent_variable_history.unprocessed_state_history` attribute
+- The dependent variables of the propagation (if any, see :ref:`dependent_variables`), stored in the
+  :attr:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults.dependent_variable_history.dependent_variable_history` attribute
+
+Each of these is returned as a dictionary, with the numerical integration epochs as key, and the current state/dependent variable
+as value. For example:
+
+.. code-block:: python
+
+    # Propagate dynamics
+    dynamics_simulator = numerical_simulation.create_dynamics_simulator(
+        body_system, propagator_settings)
+
+    # Retrieve object with full results
+    propagation_results = dynamics_simulator.propagation_results
+
+    # Retrieve the individual result histories
+    states = propagation_results.state_history
+    unprocessed_states = propagation_results.unprocessed_state_history
+    dependent_variables = propagation_results.dependent_variable_history
+
+The order of the entries of the state/dependent variable vector at every epoch can be printed to the consol before propagation as
+described :ref:`here <console_output>`. This information can also be retrieved as dictionaries from the
+:attr:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults.state_ids`
+:attr:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults.unprocessed_state_ids` and
+:attr:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResults.dependent_variable_ids`
+attributes.
+
+Additional results of the propagation, such as the runtime (e.g. real clock time) and number of function
+evaluations as a function of simulation time  (e.g. epoch in th simulation) can also be extracted from
+the :class:`~tudatpy.numerical_simulation.propagation.SingleArcPropagatorResult` class.
```

Comment:
If this can also be extracted, maybe link to the API where all the options are layed out (if that exists)